sigal/plugins/feeds.py

```diff
@@ -31,35 +31,38 @@
 
 
 def generate_feeds(gallery):
-    # Get all images and sort by date
+    # Get all images and videos and sort by date
     images = [img for album in gallery.albums.values()
               for img in album.images if img.date is not None]
-    images.sort(key=lambda im: im.date, reverse=True)
+    videos = [vid for album in gallery.albums.values()
+              for vid in album.videos if vid.date is not None]
+    medias = images + videos
```

Comment:
You can use directly `album.medias`.  (`album.images` and `album.videos` are the filtered versions of the first one).